code/addons/interactions/src/preset/preview.ts

```diff
@@ -37,7 +37,7 @@ const addSpies = (id: string, val: any, key?: string): any => {
     if (Array.isArray(val)) {
       return val.map((item, index) => addSpies(id, item, `${key}[${index}]`));
     }
-    if (typeof val === 'function' && val.name === 'actionHandler') {
+    if (typeof val === 'function' && val.isAction) {
```

Comment:
We should get @ghengeveld to check this change.

Reply:
How about either/or?